Directory.Packages.props

```diff
@@ -5,25 +5,23 @@
     <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
   </PropertyGroup>
   <ItemGroup Label="Latest DotNet Package Versions. AutoUpdate" Condition="'$(TargetFramework)' == '$(LatestSupportedDotNetVersion)' OR '$(IsNetStandard)'">

```

Comment:
latest shouldn't really have PreserveMajor defined if we want to follow an automated stay-current model.
Otherwise every major version bump will require manually curating every package version.

Reply:
I'd prefer manually updating once a year after vNext release over manually downgrading each package update PR between release and adoption.